tests/unit/components/loader/Loader.spec.jsx

```diff
@@ -0,0 +1,16 @@
+import Loader from '~/components/loader/Loader.jsx'
```

Comment:
These changes are already included in this PR #111. Could you please revert then or change the base branch from develop to #111 ?

Reply:
thanks. fixed!